tools/test-proxy/documentation/using-in-devops.md

```diff
@@ -52,3 +58,14 @@ If you're a bit confused as to what is meant by `<lowercase-language>`, refer to
 A bunch of the "common" functions are defined and called this way.
 
 This function will be _automatically picked up_ during all your builds once it's merged! How? Check in [eng/common/scripts/trust-proxy-certificate.ps1](../../../eng/common/scripts/trust-proxy-certificate.ps1) if you wish to understand.
+
+## Error Investigations
+
+When diagnosing failures that "only occur in CI" or on platforms where there is no way to get an actual debugging session, leverage the `debug` logging level. To take advantage of this in CI, create a build variable when queuing your build.
+
+![image](https://user-images.githubusercontent.com/45376673/153307363-521d271c-980e-425b-876a-212fb1e5e7a3.png)
```

Comment:
Should we save these screenshots in the repo near the .md file?

Reply:
I don't for any of the other images :P